### PR TITLE
Add support for updating buildComposerProject's vendorHash

### PR DESCRIPTION
**Pull request by gaelreyrol**
Hi,

I hope this PR is fine, I mostly did copy/paste pasta from other pull requests and it worked but I may have missed some things.

I adds the support for `buildComposerProject` which allows to build PHP projects: https://nixos.org/manual/nixpkgs/stable/index.html#ssec-building-php-projects.

Kind regards :)

### CODE REVIEW

File: tests/testpkgs/default.nix [hunk 1]
```diff
@@ -4,6 +4,7 @@
   bitbucket-snapshot = pkgs.callPackage ./bitbucket.nix { isSnapshot = true; };
   cargoLock.expand = pkgs.callPackage ./cargo-lock-expand { };
   cargoLock.update = pkgs.callPackage ./cargo-lock-update { };
+  composer.update = pkgs.callPackage ./composer.nix { };
```
Comment (Mic92) [inline tests/testpkgs/default.nix:7]:
```suggestion
  composer = pkgs.callPackage ./composer.nix { };
```

Comment (Mic92) [reply tests/testpkgs/default.nix]: The test is updating "composer" and not "composer.update"

**Comment by Mic92** [comment]
Merged in https://github.com/Mic92/nix-update/pull/217

File: tests/testpkgs/default.nix (hunk 1 above)
Comment (gaelreyrol) [reply tests/testpkgs/default.nix]: Oh thanks!

**Pull request closed without merging**